### ISSUE TITLE
Project doesn't compile out of the box

### ISSUE THREAD

**Issue by cyberoblivion** [spring-guides/gs-messaging-reactor#10]
If you clone the repository and try to build the complete project the build will fail with
-------------------------------------------------------------
hello/Receiver.java:[23,49] cannot access reactor.core.alloc.Recyclable
  class file for reactor.core.alloc.Recyclable not found

It starts to work if you remove the explicit version of the reactor
    <dependency>
      <groupId>io.projectreactor</groupId>
      <artifactId>reactor-bus</artifactId>
      <version>2.0.2.RELEASE</version>
    </dependency>

**Comment by gregturn** [comment]
The reactor guides are way behind. We need Stephane to revise them. Probably won't happen at least until after the holidays.

**Comment by thomasbittermann** [comment]
this patch did it for me:
```
diff --git a/initial/build.gradle b/initial/build.gradle
index 53f7203..9f5e366 100644
--- a/initial/build.gradle
+++ b/initial/build.gradle
@@ -4,6 +4,7 @@ buildscript {
   }
   dependencies {
     classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.2.RELEASE")
+    classpath('io.spring.gradle:dependency-management-plugin:0.5.4.RELEASE')
   }
 }
 
@@ -28,7 +29,7 @@ dependencies {
   compile "org.springframework.boot:spring-boot-starter",
       "org.springframework:spring-web",
       "com.fasterxml.jackson.core:jackson-databind",
-      "io.projectreactor:reactor-bus:2.0.2.RELEASE"
+      "io.projectreactor:reactor-bus"
 
   testCompile("junit:junit")
 }
```

**Comment by gregturn** [comment]
I appreciate you finding a work around, but the reactor-specific guides are will likely be replaced with something more demonstrative of reactive programming soon. Stay tuned!

**Comment by thomasbittermann** [comment]
... sounds great - looking forward!

**Comment by cyberoblivion** [comment]
Maybe someone should update the home page to let people know this is deprecated.

**Comment by shakhal** [comment]
+1

**Comment by vivsrivastav** [comment]
Hi @gregturn, 
Even if i fix the compilation issue by moving to the latest (2.0.7.RELEASE) of reactor-bus this code sample does not execute the Receiver flow. 
Is there any tentative timeframe by when to expect the updated documentation about reactor eventbus?  
By any chance is this example also irrelevant now? https://github.com/spring-guides/gs-reactor-thumbnailer

**Comment by dsyer** [comment]
I fixed the compilation problem (#7). @smaldini will have to help with the functionality (and that can be a new issue).
